src/commands/consumers.ts

```diff
@@ -58,6 +58,9 @@ export class StartConsumerCommandHandler {
                 preserveFocus: true,
                 viewColumn: vscode.ViewColumn.Beside,
             });
+        // Force the document language to kafka-consumer to benefit with syntax coloration
```

Comment:
Just a small thing - I would understand it well from the API call, so comment might not be necessary

Reply:
Fixed, I removed the comment.